superset/utils/cache.py

```diff
@@ -27,8 +27,9 @@ def view_cache_key(*args: Any, **kwargs: Any) -> str:  # pylint: disable=unused-
 
 
 def memoized_func(
-    key: Callable = view_cache_key, attribute_in_key: Optional[str] = None
-) -> Callable:
+    key: Callable[..., str] = view_cache_key,  # pylint: disable=bad-whitespace
```

Comment:
Note I'm unsure why `pylint` is barfing here. Per SIP-46 there's tentative plans to fix all the `pylint` issues (including bumping the version) and I sense for now we should  just accept the `pylint` "regressions".